manifests/init.pp

```diff
@@ -35,6 +35,7 @@
   String                                  $user,
   Hash                                    $virtualhost_defaults,
   Hash                                    $virtualhosts,
+  Optional[Array[String]]                 $disco_items  = undef,
```

Comment:
instead of Optional, can you please default to an empty array?
```suggestion
  [Array[String[1]]]                      $disco_items  = [],
```

Reply:
solved in 91da1e311bb6e945937bca205d38e033635f574e